utils/index.d.ts

```diff
@@ -995,7 +997,7 @@ export interface IAzureQuickPickItem<T = undefined> extends QuickPickItem {
 /**
  * Provides additional options for QuickPicks used in Azure Extensions
  */
-export interface IAzureQuickPickOptions extends QuickPickOptions, AzExtUserInputOptions {
+export interface IAzureQuickPickOptions extends VSCodeQuickPickOptions, AzExtUserInputOptions {
```

Comment:
I think in the past, we used to prefix all of our interfaces/classes/etc with `Azure`, but decided to change it to `AzExt` due to there being confusion about using the `Azure` interfaces on non-Azure resources.

This came about specifically when we started working with local function projects and the name `AzureParentTreeItem` didn't really make sense on it.

So unless something actually has a hard dependency on Azure (as in, it requires an Azure subscription/resource), we have been opting to prefix it with `AzExt`. I personally prefer this naming scheme, but no strong opinions either way.

Reply:
I don't wanna change this now, though. We aren't adding the interface, it's already in.

Reply:
Oh, whoops. Okay, probably back from when we were still using `Azure` as the prefix then 😬

Reply:
I agree that we should change it. But also agree we should do it later.